data/hooks/encoding.py

```diff
@@ -15,6 +15,27 @@
 
 @registered_data_hook("one_hot_encode")
 class OneHotEncoding(FittedDataHook):
+    """
+    Modified implementation of SciKit-Learn's OneHotEncoder to allow it to function in the context of data of mixed
+        categorical/continuous features. Which features it will be applied to can be specified with two configuration
+        parameters:
+
+    * `features`: A list of features within the dataset which should explicitly be treated as categorical and encoded
+        using this data hook
+    * `max_unique_vals`: An integer threshold; if a feature within the dataset has fewer than this many unique
+        values across all samples is assumed to be "categorical", and is encoded.
+
+    How the resulting data is encoded is otherwise identical to SciKit's implementation; its parameters can be specified
+    like any other config option (see "handle_unknown" in the example below).
+
+    Example usage:
+    {
+        "type": "one_hot_encode",
+        "features": ["color"],
+        "max_unique_vals": 5,
+        "handle_unknown: "ignore"
+    }
```

Comment:
Maybe we could also add an example for a binary variable?
```suggestion
    Example usage:
    {
        "type": "one_hot_encode",
        "features": ["color"],
        "max_unique_vals": 5,
        "handle_unknown: "ignore"
    }
    In the case of a binary variable:
        {
        "type": "one_hot_encode",
        "features": ["sex"],
        "drop": "if_binary",
        "handle_unknown": "warn"
    }
```